src/ol/PluggableMap.js

```diff
@@ -604,7 +604,7 @@ class PluggableMap extends BaseObject {
     }
     const options = opt_options || /** @type {AtPixelOptions} */ ({});
```

Comment:
Not related to your change here, but I think we should also address type casts like the one above.  If an empty object is not assignable to `AtPixelOptions`, we shouldn’t be using it as one.

Reply:
You're right: these typecast are not necessary. Typescript is able to deduce that an empty object is compatible with `AtPixelOptions`.

pull request to come.

Reply:
See #10076